asm/melee/cm/camera.s

```diff
@@ -9219,7 +9219,7 @@ func_80030CFC:
 /* 80030D44 0002D924  38 81 00 50 */	addi r4, r1, 0x50
 /* 80030D48 0002D928  38 BE 00 1C */	addi r5, r30, 0x1c
 /* 80030D4C 0002D92C  38 C1 00 38 */	addi r6, r1, 0x38
-/* 80030D50 0002D930  4B FD DA E9 */	bl func_8000E838
+/* 80030D50 0002D930  4B FD DA E9 */	bl lbvector_some_camera_vector_computation
```

Comment:
I'm not sure calling this a camera vector computation is any more descriptive and is potentially misleading if it's used elsewhere.

Just stick to lbvector_8000E838